examples/helpers/android/app/CMakeLists.txt.in

```diff
@@ -53,6 +54,11 @@ add_custom_command(
         ${CMAKE_CURRENT_SOURCE_DIR}/libs/${EDGE_SDK_NETWORK_PROTOCOL_JAR_NAME}
 )
 
+target_include_directories(dataservice-read-example
```

Comment:
```suggestion
target_include_directories(@EDGE_SDK_EXAMPLE_TARGET_NAME@
```
This won't work for `dataservice-write` target, so you have to use generated target name.